crates/timber/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+mod formatter;
+pub use tracing_core::Level;
+
+/// possible log levels
+pub const LEVELS: [&str; 5] = ["error", "warn", "info", "debug", "trace"];
+
+#[cfg(debug_assertions)]
```

Comment:
Just so I'm understanding right this means that in non-release mode, the default level is `debug` while in release mode, it's `info`, right?

Reply:
yes! that is correct